### DISCUSSION TITLE
How to send StatusNoContent without sending content type?

### DISCUSSION THREAD

**Discussion by OrkhanAlikhanov** [goravel/goravel#274] [Q&A]
After deleting an entity, I'd like to send `http.StatusNoContent` with no body. How do I achieve it?

I tried:
```go
return ctx.Response().Json(http.StatusNoContent, nil)
```

**Comment by OrkhanAlikhanov** [comment]
Workaround:
```go
ctx.Response().Writer().WriteHeader(http.StatusNoContent)
return nil
```

**Reply by hwbrzzl** [reply]
Yes, we may add a new `NoContent()` method.

**Reply by hwbrzzl** [reply]
Created an issue: https://github.com/goravel/goravel/issues/275
